File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop5.java
package sample;
import java.util.*;
 class Forloop5 
 {
  
  public static void main (String[] ar)
  {
  int n,i,j;
  double z,sum=0;
  System.out.println("enter the range");
  Scanner m=new Scanner(System.in);
  n=m.nextInt();
  for(i=1;i<=n;i++)
  {
    
    System.out.print(+i+"^"+i);
    if(i<n)
    {
      System.out.print(" + ");
    }
    z=Math.pow(i,i);
    sum=sum+z;
  }
  System.out.print("sum of seriese="+sum);   
    
  }
  
}

File: python programming/forloop_5.py
n=int(input("enter no of term:-"))
k=1
sum=0
for i in range(1,n+1):
 print(f'{i}/{k}*{k+1}',end='')
 if i<n:
  print(" + ",end='')
 sum=sum+i/(k*(k+1))
 k=k+2
print("sum of seriese=",sum)
 
 
 